docker/fullnode/README.md

```diff
@@ -1,83 +1,93 @@
-# run a sui devnet fullnode locally
+# Run a Sui DevNet Fullnode Locally with Docker
 
-Run a Sui DevNet fullnode locally for testing/experimenting by following the instructions below. This has been tested and should work for:
+Run a Sui DevNet [fullnode](../../doc/src/build/fullnode.md) locally for testing/experimenting by following the instructions below. This has been tested and should work for:
 
 - linux/amd64
 - darwin/amd64 
 - darwin/arm64
 
-# prerequisites
+## Prerequisites
 
-Install docker / docker compose:
+Install Docker / Docker Compose:
 - https://docs.docker.com/get-docker/
 - https://docs.docker.com/compose/install/
 
-# run
+## Running
 
-## fullnode config
+### Fullnode config
 
-Get the latest version of the fullnode config [here](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml), or:
+Download the latest version of the fullnode config [fullnode-template.yaml](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml) over the web or by using `curl` or `wget`, for example:
 
-```wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml```
+```shell
+wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml
+```
+
+TODO: Don't they need to rename this file to `fullnode.yaml` too?
```

Comment:
Technically no as I configured it to require the least steps possible to get up and running, see: 

https://github.com/MystenLabs/sui/blob/e03a0af538d6d70a9744012438192950ea7d5599/docker/fullnode/docker-compose.yaml#L13

Reply:
If we want consistency we can add a step to rename/move the template prior to starting docker-compose however if its not necessary then i prefer one less step.

Reply:
Perfect. Thank you!

Reply:
Removed TODO.